test/functional/api/cas/casadm.py

```diff
@@ -297,9 +297,7 @@ def get_param_promotion_nhit(
 def set_cache_mode(
     cache_mode: CacheMode, cache_id: int, flush: bool = None, shortcut: bool = False
 ) -> Output:
-    flush_cache = None
-    if flush is not None:
-        flush_cache = "yes" if flush else "no"
+    flush_cache = "yes" if flush else "no"
```

Comment:
Option is required when switching from Write-Back or Write-Only mode
In other calls can be None